security_groups.tf

```diff
@@ -96,65 +96,65 @@ resource "aws_security_group_rule" "ingress_internet_proxy" {
   security_group_id        = data.terraform_remote_state.internal_compute.outputs.internet_proxy.sg
 }
 
-resource "aws_security_group_rule" "egress_hbase_zookeeper" {
-  description              = "Ingest-HBase Zookeeper requests"
-  type                     = "egress"
-  from_port                = 2181
-  to_port                  = 2181
-  protocol                 = "tcp"
-  source_security_group_id = data.terraform_remote_state.ingest.outputs.emr_common_sg.id
-  security_group_id        = aws_security_group.bgdc_common.id
-}
+//resource "aws_security_group_rule" "egress_hbase_zookeeper" {
```

Comment:
Since this is going to production [?] its worth removing these resources entirely rather than commenting them out else it can cause confusion